src/main/java/me/skulduggerry/gameoflife/controller/Frontend.java

```diff
@@ -1,15 +1,18 @@
 package me.skulduggerry.gameoflife.controller;
 
+import static me.skulduggerry.gameoflife.controller.Binding.*;
```

Comment:
Do not use `*` imports. Import the desired classes directly. This can be configured in IntelliJ via:
Settings > Editor > Code Style > Java > Imports:
Class counts to use import with `*`: 9999
Names counts to use static import with `*`: 9999

![image](https://user-images.githubusercontent.com/18695858/150388243-dd4a168a-6b38-4cbd-b425-d8cca6845e7a.png)